package.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "aqa-javascript-advanced",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Add your command here
```suggestion
    "test": "node js-operators-coercion/operatorsAndCoercion.js"
```

and run your tests as : npm run test